docs/source/user/configuration.rst

```diff
@@ -25,7 +25,8 @@ Configuration Locations
 =======================
 
 |Flake8| supports storing its configuration in your project in one of
```

Comment:
it already says "one of"

Reply:
It is correct, it's just a little ambiguous. My first thought after reading original sentence was which one will be used if there is part of configuration in all of the files. I end up looking for answer in source code and here on github.